yt_dlp/extractor/canvas.py

```diff
@@ -276,35 +276,49 @@ def _login(self):
         if username is None:
             return
 
-        auth_data = {
-            'APIKey': self._APIKEY,
-            'targetEnv': 'jssdk',
-            'loginID': username,
-            'password': password,
-            'authMode': 'cookie',
-        }
-
-        auth_info = self._gigya_login(auth_data)
+        # 1. Get Login Data
+        try:
+            auth_info = self._download_json(
+                'https://accounts.vrt.be/accounts.login', None,
+                note='Login data', errnote='Could not get Login data',
+                headers={}, data=urlencode_postdata({
+                    'loginID': username,
+                    'password': password,
+                    'sessionExpiration': '-2',
+                    'APIKey': self._APIKEY,
+                    'targetEnv': 'jssdk',
+                }))
+        except ExtractorError as e:
+            self.report_warning("Cookie request HTTP code: %s" % e.cause.code)
+            raise e
```

Comment:
why?

Reply:
ok, removed in the latest ytdl-org/youtube-dl#29614